src/main/java/com/openshift/internal/restclient/model/volume/VolumeSource.java

```diff
@@ -14,22 +14,22 @@
 
 import java.util.HashMap;
 
+import com.openshift.restclient.model.volume.VolumeType;
 import org.jboss.dmr.ModelNode;
 
 import com.openshift.internal.restclient.model.ModelNodeAdapter;
 import com.openshift.internal.restclient.model.properties.ResourcePropertyKeys;
-import com.openshift.restclient.model.volume.IVolume;
 import com.openshift.restclient.model.volume.IVolumeSource;
 
 /**
  * 
  * @author Jeff Cantrill
  *
  */
-public class VolumeSource 
+public abstract class VolumeSource
```

Comment:
This change will depend on the diff between whats in this object and whats returned from the REST operations.  i would hope they are the same but if not then please ignore this comment until we can further evaluate it.